test/model/test_node.py

```diff
@@ -79,6 +79,25 @@ async def test_set_value(node, uuid4, mock_command):
     }
 
 
+async def test_poll_value(node, uuid4, mock_command):
+    """Test poll value."""
+    ack_commands = mock_command(
+        {"command": "node.poll_value", "nodeId": node.node_id},
+        {"result": "something"},
+    )
+    value_id = "52-32-00-currentValue-00"
+    value = node.values[value_id]
+    assert await node.async_poll_value(value_id)
```

Comment:
Should we check the return value more specifically? We know what it will be in this case.

Reply:
done